viewer/package.json

```diff
@@ -8,7 +8,7 @@
     "release": "npm run mkdist && webpack",
     "build": "npm run mkdist && webpack --env.development",
     "mkdist": "run-script-os",
-    "mkdist:win32": "if not exist 'dist' mkdir 'dist' && copy package.json README.md dist",
+    "mkdist:win32": "@powershell $(if(-not(Test-Path dist -PathType Container)) { mkdir dist;$? } else { $true;$? }) -and $(Copy-Item -Path package.json, README.md -Destination dist;$?)",
```

Comment:
Oh god, is this really an improvement? Powershell gives me the chills

Reply:
We might want to just write this logic in JS and have it the same on all OSes at some point.

Or in Rust :smile:

Reply:
It is not an improvement, but I am using powershell in VSCode, and my terminals open as such by default. The run-script-os should be able to see if the shell running the script is powershell or cmd, then it could be like win32-cmd, win32-ps, that would be cool 😅